custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -1109,7 +1115,7 @@ def possibly_intuit_order(
             )
             print(e, file=sys.stderr)
             raise DrumCommonException(e)
-        uniq = df[target_col_name].sample(min(1000, len(df)), random_state=1).unique()
+        uniq = df[target_col_name].unique()
```

Comment:
did we want to sort these?

Reply:
They weren't sorted before this PR.  Is there a reason to?

Reply:
so the order doesn't change around between runs...

Reply:
Sort has been added.  Can you approve?